assets/dev/js/editor/controls/behaviors/scrubbing.js

```diff
@@ -95,16 +99,14 @@ export default class Scrubbing extends Marionette.Behavior {
 		const checkIntentTimeout = setTimeout( () => {
 			clearTimeout( checkIntentTimeout );
 			document.addEventListener( 'mousemove', trackMovement );
-
-			document.body.classList.add( this.scrubSettings.scrubbingClass );
+			$e.uiStates.set( 'document/scrubbing-mode', 'on' );
```

Comment:
Maybe add some constant?
I just don't like seeing string just hanging there...

```suggestion
			$e.uiStates.set( 'document/scrubbing-mode', ScrubbingMode.ON );
```

Reply:
Done.